de_web_plugin.cpp

```diff
@@ -1921,6 +1921,10 @@ void DeRestPluginPrivate::gpDataIndication(const deCONZ::GpDataIndication &ind)
                 {
                     sensorNode.setName(QString("FoH Switch %2").arg(sensorNode.id()));
                 }
+                else if (sensorNode.Manufacturer() == QLatin1String("Legrand"))
```

Comment:
When compiling on my armv7l board, I had to lowercase "sensorNode.**m**anufacturer()".

<img width="938" alt="2021-07-13-14-25-16 iTerm2 frederic@opi ~deconz-rest-plugin" src="https://user-images.githubusercontent.com/954168/125451467-6c424864-a93b-4a89-818c-1d891406be61.png">

Reply:
corrected